app/helpers/reservations_helper.rb

```diff
@@ -42,15 +42,14 @@ def bar_span_positioning_fix
   def manage_reservations_btn # rubocop:disable all
     return if (cannot? :manage, Reservation) || @reservation.reserver.id.nil?
     if (can? :override, :reservation_errors) &&
-       @reservation.approval_status == 'requested'
+       @reservation.flags & Reservation::FLAGS[:request]
```

Comment:
Awesome bitmasking! but I think it'll be more readable and less prone to screwing up if we wrap it in a higher level model method to access flags?
```
def flag(flag_type)
  return self.flags  & Reservation::FLAGS[flag_type]
end
```
or something